### DISCUSSION TITLE
Multi drop increase and decrease!!

### DISCUSSION THREAD

**Discussion by MLastAirBender** [goToMain/libosdp#183] [General]
Is there any way to increase the multi drop client from 2 to 4 during run time?
I don't want to reset or stop the things which are running but want to increase the number of readers on the same port.

**Comment by sidcha** [comment]
As discussed in email, this can be achieved with a new API to enable and disable PDs. Will create an issue to track that change.

**Comment by MLastAirBender** [comment]
Thank you @sidcha

**Comment by MLastAirBender** [comment]
Hi @sidcha , can direct me to the branch or API where you made the required changes?

**Reply by sidcha** [reply]
I haven't gotten around to this yet. You can watch the tracking issue #184 for when this change lands in master.
